test/others/test_utils.py

```diff
@@ -199,14 +199,14 @@ def test_calculate_context_similarity_on_partially_overlapping_contexts_with_noi
     assert accuracy > 0.99
 
 
-def test_match_context():
-    whole_document = TEST_CONTEXT
+def test_match_context_multi_process():
+    whole_document = TEST_CONTEXT[:300]
```

Comment:
Just to be sure, this is still relevant after you fixed the dependency version?

Reply:
Yes, without the `pool.join()` there are nasty errors in the log. With `pool.join()` the coming loop takes much longer and it makes just more sense to have it this way (with lots of candidates instead of lots of iterations).